ocos/kernels/sentencepiece_tokenizer.cc

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "sentencepiece_processor.h"
+#include "sentencepiece_model.pb.h"
+#include "sentencepiece_tokenizer.hpp"
```

Comment:
Can it be moved into the tokenizer folder?
Given each tokenizer is large, the cmake option ENABLE_TOKENIZER may be renamed to ENABLE_GPT2_TOKENIZER